### PR TITLE
Improve .emoji

### PR DESCRIPTION
**Pull request by iandioch**
Changes

- "aubergines" (-s plurals) will be recognised as a form of "aubergine", and will map correctly to the emoji.
- "peaches" (-es plurals) will do the same.
- verbs ending in "ing" will drop the "ing" if that will match an emoji.
- "it" and "is" and others will no longer map to flag emojis.
- the output is better spaced, and is more varied in positioning and number of emojis.

### CODE REVIEW

File: steely/plugins/emoji.py [hunk 1]
```diff
@@ -28,37 +27,54 @@
...
+    w = word.lower()
+    w = ''.join(c for c in w if c.isalnum())
+    if not len(w):
+        return word
+    if w not in WORD_TO_EMOJI and w[-2:] == 'es':
+        w = w[:-2] # peaches -> peach
+    if w not in WORD_TO_EMOJI and w[-1] == 's':
+        w = w[:-1] # aubergines -> aubergine
+    if w not in WORD_TO_EMOJI and w[-3:] == 'ing':
+        w = w[:-3] # hating -> hat
```
Comment (CianLR) [inline steely/plugins/emoji.py]: I would probably smack these in a list and use `w.startswith(thing)`

Comment (iandioch) [reply steely/plugins/emoji.py]: .mock

Comment (iandioch) [reply steely/plugins/emoji.py]: I think putting these in a list and doing w.endswith would be less clear. They all handle specific cases (and there were other cases I didn't bother adding next to them. Like I was debating "running" -> "runn" -> "run", but cba) that are more clearly laid out like this. Plus, the order they are run is actually important (ie. removing "s" from "peaches" would be useless), and putting them in a list wouldn't make it as clear not to mix them up later.

Ideally, I want to do some real NLP to map plurals to singulars (so "men" becomes "man" too), but that will come later.

**Comment by iandioch** [comment]
Closes #157.

**Pull request merged**